test/devices_tests/fan_test.py

```diff
@@ -60,6 +60,42 @@ async def test_sync_state_address(self):
         )
 
     #
+    # TEST SWITCH ON/OFF
+    #
+    async def test_swith_on_off(self):
+        """Test switching on/off of a Fan."""
+        xknx = XKNX()
+        fan = Fan(xknx, name="TestFan", group_address_switch="1/2/3")
+
+        # Turn the fan on
+        await fan.turn_on()
+        assert xknx.telegrams.qsize() == 1
+        telegram = xknx.telegrams.get_nowait()
+        assert telegram == Telegram(
+            destination_address=GroupAddress("1/2/3"),
+            payload=GroupValueWrite(DPTBinary(1)),
+        )
+
+        # Turn the fan off
+        await fan.turn_off()
+        assert xknx.telegrams.qsize() == 1
+        telegram = xknx.telegrams.get_nowait()
+        assert telegram == Telegram(
+            destination_address=GroupAddress("1/2/3"),
+            payload=GroupValueWrite(DPTBinary(0)),
+        )
+
+        # Turn the fan on again this time with a provided speed, which however has no
+        # additional effect, since in the switch GA mode, the speed during switching on
+        # will not be considered since that is controlled independently in this mode
```

Comment:
That might lead to a bit of confusion, since this behavior is not completely obvious, so needs to be documented in the HA KNX docs. Otherwise users might be surprised that in their service calls the percentage/speed gets ignored, although it completely makes sense if they configure a fan with a dedicated switch GA.

Reply:
With light entities we have a similar situation - this could indeed be somewhat confusing. We don't send to a binary GA when receiving a turn_on call with brightness value there. 
But for fans there is the additional FAN.SET_PERCENTAGE service so this could be ok.